test/integration/good/function-signatures/distributions/univariate/continuous/logistic/logistic_lpdf.stan

```diff
@@ -143,735 +143,735 @@ parameters {
 }
 transformed parameters {
   real transformed_param_real;
-  transformed_param_real = logistic_lpdf(d_int | d_int, d_int);
```

Comment:
It looks like you've replace the `logistic_lpdf` signature tests with `loglogistic_lpdf` tests, shouldn't the  `loglogistic_lpdf` be tested as well as, not instead of?

Reply:
Ah, seems I was sloppy with these tests. What I was trying to achieve was to copy the logistic folder and then find and replace the calls. Thanks for finding this out.

Reply:
Should be fixed now.